lib/winston-dynamodb.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+var winston = require('winston');
+var util = require('util');
+var AWS = require('aws-sdk');
+var uuid = require('node-uuid');
+var _ = require('lodash');
+var os = require('os');
+var hostname = os.hostname();
+function datify(timestamp) {
+    var dateTS = new Date(timestamp);
+    var date = {
+        year: dateTS.getFullYear(),
+        month: dateTS.getMonth() + 1,
+        day: dateTS.getDate(),
+        hour: dateTS.getHours(),
+        minute: dateTS.getMinutes(),
+        second: dateTS.getSeconds(),
+        millisecond: dateTS.getMilliseconds()
+    };
+    var keys = _.without(Object.keys(date), "year", "month", "day");
```

Comment:
NOTE; I'm not sure this was correct originally, so I tweaked it a bit to get it to compile.